cli/operator/node.go

```diff
@@ -594,6 +596,21 @@ func setupSSVNetwork(logger *zap.Logger) (networkconfig.NetworkConfig, error) {
 		return networkconfig.NetworkConfig{}, err
 	}
 
+	if cfg.SSVOptions.CustomDomainType != "" {
+		if !strings.HasPrefix(cfg.SSVOptions.CustomDomainType, "0x") {
+			return networkconfig.NetworkConfig{}, errors.New("custom domain type must be a hex string")
+		}
+		byts, err := hex.DecodeString(cfg.SSVOptions.CustomDomainType[2:])
+		if err != nil {
+			return networkconfig.NetworkConfig{}, errors.Wrap(err, "failed to decode custom domain type")
+		}
+		if len(byts) != 4 {
+			return networkconfig.NetworkConfig{}, errors.New("custom domain type must be 4 bytes")
+		}
+		networkConfig.GenesisDomainType = spectypes.DomainType(byts)
+		logger.Info("running with custom domain type", fields.Domain(networkConfig.GenesisDomainType))
```

Comment:
should probably add something like:
```suggestion
if networkConfig.PastAlanFork() {
    logger.Info("custom domain type is ineffective now after Alan fork", fields.Domain(networkConfig.GenesisDomainType))
} else {
    logger.Info("running with custom domain type", fields.Domain(networkConfig.GenesisDomainType))
}
```